contrib/puppetlabs-lumogon/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Change log
+
+All notable changes to this project will be documented in this file.
+
+## 0.4.0 (June 8, 2017)
```

Comment:
My thought here is that because this is part of the same repo, they'd follow the same tagging strategy. (e.g.: v0.4.0 of Lumogon works with the same versioned Puppet Module). I started to reason about the tagging strategy for the module as it related to the current build process and saw that I could either:

* Remove this to it's own module and let it follow it's own version path, or
* Line up the versions so a tag on the repo lines up appropriately.

It may still be better in the long run to move this to it's own repo if we can't line up the automation, but either way that's the rationale